packages/components/src/components/menusection/menusection.stories.ts

```diff
@@ -0,0 +1,61 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit';
+import { classArgType, styleArgType } from '../../../config/storybook/commonArgTypes';
+import { disableControls } from '../../../config/storybook/utils';
+import '.';
+import '../menuitem';
+import '../menuitemcheckbox';
+import '../menuitemradio';
+import '../divider';
+
+const wrapWithDiv = (htmlString: TemplateResult) => html`<div role="menubar" style="width: 25rem;">${htmlString}</div>`;
+
+const render = () => wrapWithDiv(html`
+    <mdc-menusection>
+      <mdc-menuitemcheckbox label="Checkbox1" aria-label="Checkbox 1">Checkbox1</mdc-menuitemcheckbox>
+      <mdc-menuitemcheckbox label="Checkbox2" aria-label="Checkbox 2">Checkbox2</mdc-menuitemcheckbox>
+      <mdc-menuitemcheckbox label="Checkbox3" aria-label="Checkbox 3">Checkbox3</mdc-menuitemcheckbox>
+      <mdc-menuitemradio label="Radio1" aria-label="Radio 1">Radio1</mdc-menuitemradio>
+      <mdc-menuitemradio label="Radio2" aria-label="Radio 2">Radio2</mdc-menuitemradio>
+      <mdc-menuitemradio label="Radio3" aria-label="Radio 3">Radio3</mdc-menuitemradio>
```

Comment:
```suggestion
      <mdc-menuitemcheckbox label="Checkbox1" aria-label="Checkbox 1"></mdc-menuitemcheckbox>
      <mdc-menuitemcheckbox label="Checkbox2" aria-label="Checkbox 2"></mdc-menuitemcheckbox>
      <mdc-menuitemcheckbox label="Checkbox3" aria-label="Checkbox 3"></mdc-menuitemcheckbox>
      <mdc-menuitemradio label="Radio1" aria-label="Radio 1"></mdc-menuitemradio>
      <mdc-menuitemradio label="Radio2" aria-label="Radio 2"></mdc-menuitemradio>
      <mdc-menuitemradio label="Radio3" aria-label="Radio 3"></mdc-menuitemradio>
```

menu items / listitems won't take text from the slot, so this will show a bad example for the consumer.
Also, you can remove aria label also, as the label and aria label have both text, the screen reader will read it 2 times.